README.md

```diff
@@ -41,12 +41,18 @@ go install github.com/nats-io/natscli/nats@v0.0.33
 
 #### macOS installation via Homebrew
 
-For macOS `brew` can be used to install the latest version:
+For macOS `brew` can be used to install the latest released version:
 
 ```nohighlight
 brew tap nats-io/nats-tools
 brew install nats-io/nats-tools/nats
 ```
+#### Windows installation via scoop
+On Windows, [scoop](https://scoop.sh) has the latest released version:
+```
+scoop bucket add extras
+scoop install extras/natscli
+```
```

Comment:
Does it automatically update or must someone update a repository? If not automatic and not in our hands I dont like calling it out here

Reply:
@ripienaar updated PR description with an answer.